### PR TITLE
Update release-notes for release-v1.12.0

### PR DESCRIPTION
**Pull request by csmbot**
Update release-notes to include features and bug fixes

### CODE REVIEW

**Review by rajkumar-palani — APPROVED** [review]
lgtm

File: content/docs/csidriver/release/powerflex.md [hunk 1]
```diff
@@ -13,25 +13,17 @@ description: Release notes for PowerFlex CSI driver
...
-- [#1237 - [BUG]: Error handling not good in node.go:nodeProbe() and other similar functions](https://github.com/dell/csm/issues/1237)
-- [#1239 - [BUG]: Changes in new release of google.golang.org/protobuf is causing compilation issues](https://github.com/dell/csm/issues/1239)
-- [#1270 - [BUG]: Missing entries for Resiliency in installation wizard template](https://github.com/dell/csm/issues/1270)
-- [#1310 - [BUG]:  CSI node pod crash after replacing OCP ingress certificate or restarting kubectl service](https://github.com/dell/csm/issues/1310)
-- [#1350 - [BUG]: Document update : PowerFlex expecting secret CR as <csm-cr-name>-config in operator ](https://github.com/dell/csm/issues/1350)
-- [#1355 - [BUG]: Indentation of secret.yaml mentioned on the csm-doc portal for powerflex driver is incorrect.](https://github.com/dell/csm/issues/1355)
-- [#1364 - [BUG]: mkfsFormatOption not working for powerflex](https://github.com/dell/csm/issues/1364)
-- [#1366 - [BUG]:  Support Minimum 3GB Volume Size for NFS in CSI-PowerFlex](https://github.com/dell/csm/issues/1366)
+- [#1448 - [BUG]: CSM-operator build fails from disk space issue](https://github.com/dell/csm/issues/1448)
```
Comment (rajkumar-palani) [inline content/docs/csidriver/release/powerflex.md:25]: Please check whether this issue is closed in the internal JIRA.

**Review by rajkumar-palani — CHANGES_REQUESTED** [review]
can you please address the comment.

File: content/docs/csidriver/release/powerflex.md (hunk 1 above)
Comment (rajkumar-palani) [reply content/docs/csidriver/release/powerflex.md:25]: Approving this after validating the internal checks.

**Review by rajkumar-palani — APPROVED** [review]
lgtm

**Review by alikdell — APPROVED** [review]

File: content/docs/csidriver/release/powerflex.md [hunk 2]
```diff
@@ -13,25 +13,17 @@ description: Release notes for PowerFlex CSI driver
...
 
+
 ### New Features/Changes
 
-- [#1359 - [FEATURE]: Add Support for OpenShift Container Platform (OCP) 4.16 ](https://github.com/dell/csm/issues/1359)
-- [#1400 - [FEATURE]: Support for Kubernetes 1.30](https://github.com/dell/csm/issues/1400)
-- [#1358 - [FEATURE]: Support for PowerFlex 4.6](https://github.com/dell/csm/issues/1358)
-- [#1397 - [FEATURE]: Observability upgrade is supported in CSM Operator](https://github.com/dell/csm/issues/1397)
+- [#1472 - [FEATURE]: Support for Kubernetes 1.31](https://github.com/dell/csm/issues/1472)
+- [#1508 - [FEATURE]:  Add Support for Kubevirt](https://github.com/dell/csm/issues/1508)
```
Comment (AronAtDell) [inline content/docs/csidriver/release/powerflex.md:20]: This should be KubeVirt not "Kubevirt" I updated the GitHub Issue. https://github.com/dell/csm/issues/1508

File: content/docs/csidriver/release/powerscale.md [hunk 3]
```diff
@@ -14,23 +14,19 @@ description: Release notes for PowerScale CSI driver
...
+
 ### New Features/Changes
 
-- [#1359 - [FEATURE]: Add Support for OpenShift Container Platform (OCP) 4.16 ](https://github.com/dell/csm/issues/1359)
-- [#1400 - [FEATURE]: Support for Kubernetes 1.30](https://github.com/dell/csm/issues/1400)
-- [#1397 - [FEATURE]: Observability upgrade is supported in CSM Operator](https://github.com/dell/csm/issues/1397)
-- [#1398 - [FEATURE]: PowerScale  OneFS 9.7 support ](https://github.com/dell/csm/issues/1398)
+- [#1472 - [FEATURE]: Support for Kubernetes 1.31](https://github.com/dell/csm/issues/1472)
+- [#1473 - [FEATURE]: Add Support for OpenShift Container Platform (OCP) 4.17](https://github.com/dell/csm/issues/1473)
+- [#1508 - [FEATURE]:  Add Support for Kubevirt](https://github.com/dell/csm/issues/1508)
```
Comment (AronAtDell) [inline content/docs/csidriver/release/powerscale.md:22]: See above comment.

File: content/docs/csidriver/release/powerstore.md [hunk 4]
```diff
@@ -13,22 +13,22 @@ description: Release notes for PowerStore CSI driver
...
 
 
+
 ### New Features/Changes
 
-- [#1359 - [FEATURE]: Add Support for OpenShift Container Platform (OCP) 4.16 ](https://github.com/dell/csm/issues/1359)
-- [#1400 - [FEATURE]: Support for Kubernetes 1.30](https://github.com/dell/csm/issues/1400)
+- [#1472 - [FEATURE]: Support for Kubernetes 1.31](https://github.com/dell/csm/issues/1472)
+- [#1473 - [FEATURE]: Add Support for OpenShift Container Platform (OCP) 4.17](https://github.com/dell/csm/issues/1473)
+- [#1508 - [FEATURE]:  Add Support for Kubevirt](https://github.com/dell/csm/issues/1508)
```
Comment (AronAtDell) [inline content/docs/csidriver/release/powerstore.md:21]: See above comment.

File: content/docs/deployment/csmoperator/release/_index.md [hunk 5]
```diff
@@ -19,20 +19,23 @@ Description: >
...
+
 ### New Features/Changes
 
-- [#1359 - [FEATURE]: Add Support for OpenShift Container Platform (OCP) 4.16 ](https://github.com/dell/csm/issues/1359)
-- [#1277 - [FEATURE]: Add Authorization upgrade is supported in CSM Operator](https://github.com/dell/csm/issues/1277)
-- [#1396 - [FEATURE]: DCM and DN client upgrade is supported in CSM operator ](https://github.com/dell/csm/issues/1396)
-- [#1397 - [FEATURE]: Observability upgrade is supported in CSM Operator](https://github.com/dell/csm/issues/1397)
+- [#1449 - [FEATURE]: Simplify the CSM Operator deployment](https://github.com/dell/csm/issues/1449)
+- [#1473 - [FEATURE]: Add Support for OpenShift Container Platform (OCP) 4.17](https://github.com/dell/csm/issues/1473)
+- [#1508 - [FEATURE]:  Add Support for Kubevirt](https://github.com/dell/csm/issues/1508)
```
Comment (AronAtDell) [inline content/docs/deployment/csmoperator/release/_index.md:27]: See above comment.

**Pull request closed without merging**